### PR TITLE
Add updates to the upgrade to 3.0 readme

### PR DESCRIPTION
**Pull request by davidhan527**

### CODE REVIEW

**Comment by davidhan527** [comment]
Hey @oreoshake . Thought I'd make these quick changes instead of opening up more issues. Let me know what you think and if the wording is okay. Thanks!

File: upgrading-to-3-0.md [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
+| schemes in source expressions                            | Schemes were not stripped                                                                                                                                                                       | Schemes are stripped by default to discourage mixed content. Setting `preserve_schemes: true` will revert to previous behavior                                               |
+| opting out of default configuration                      | `skip_before_filter :set_x_download_options_header`                                                                                                                                             | Within default block: `config.x_download_options = SecureHeaders::OPT_OUT`                                                                                                   |
```
Comment (oreoshake) [inline upgrading-to-3-0.md]: In the "before" column, setting a config value to `false` was also a way to opt-out of a header. i.e. `config.x_download_options = false`.

File: upgrading-to-3-0.md [hunk 2]
```diff
@@ -3,17 +3,19 @@
...
+| CSP Nonce values in views                                | `@content_security_policy_nonce`                                                                                                                                                                | `content_security_policy_script_nonce` or `content_security_policy_style_nonce`                                                                                              |
+| nonce is no longer a source expression                   | `config.csp = "'self' 'nonce'"`                                                                                                                                                                 | Remove `'nonce'` from source expression and only use view helpers above                                                                                                      |
```
Comment (oreoshake) [inline upgrading-to-3-0.md]: Please add a link to the [nonce docs](https://github.com/twitter/secureheaders#nonce) here. The view helpers are preferred, but they aren't the only way to use nonces.

**Comment by oreoshake** [comment]
Thanks @davidhan527! Just two small notes.

PRs are always preferred over issues, much appreciated :smile:

**Comment by davidhan527** [comment]
Thanks for the feedback! Changes made.

**Pull request merged**